examples/variational_autoencoder_deconv.py

```diff
@@ -32,16 +32,21 @@
 epochs = 5
 
 x = Input(batch_shape=(batch_size,) + original_img_size)
-conv_1 = Convolution2D(img_chns, 2, 2, border_mode='same', activation='relu')(x)
-conv_2 = Convolution2D(filterss, 2, 2,
-                       border_mode='same', activation='relu',
-                       subsample=(2, 2))(conv_1)
-conv_3 = Convolution2D(filterss, num_conv, num_conv,
-                       border_mode='same', activation='relu',
-                       subsample=(1, 1))(conv_2)
-conv_4 = Convolution2D(filterss, num_conv, num_conv,
-                       border_mode='same', activation='relu',
-                       subsample=(1, 1))(conv_3)
+conv_1 = Conv2D(img_chns,
+                kernel_size=(2, 2),
+                padding='same', activation='relu')(x)
+conv_2 = Conv2D(filters,
+                kernel_size=(2, 2),
+                padding='same', activation='relu',
+                strides=2)(conv_1)
```

Comment:
If using a tuple as the `kernel_size` argument, better to use a tuple for `strides` as well, for consistency.